ani-cli

```diff
@@ -150,54 +199,69 @@ get_video_quality() {
 	printf '%s' "$video_link"
 }
 
-dep_ch () {
-	for dep; do
-		if ! command -v "$dep" >/dev/null ; then
-			die "Program \"$dep\" not found. Please install it."
-		fi
-	done
+
+###############
+# Text output #
+###############
+
+err () {
+	# display an error message to stderr (in red)
+	printf "$c_red%s$c_reset\n" "$*" >&2
 }
 
-# get query
-get_search_query () {
-	if [ -z "$*" ]; then
-		printf "$c_cyan%s" "Search Anime: "
-		read -r query
-	else
-		query=$*
-	fi
+inf () {
+	# display an informational message (first argument in green, second in magenta)
+	printf "$c_green%s $c_magenta%s$c_reset\n" "$1" "$2"
 }
 
-# create history file
-[ -f "$logfile" ] || : > "$logfile"
+prompt () {
+        # prompts the user with message in $1 and saves the input to the variables in $2 and $3 
+        printf "$c_blue%s: $c_reset" "$1"
+        read -r input1 input2
+        # indirectly assigning the input to the variable in $2 and in $3 if present
+        if [ -z "$3" ]; then
+                eval "$2=\$input1"
+        else
+                eval "$2=\$input1"
+                eval "$3=\$input2"
+        fi
+}
 
-#####################
-## Anime selection ##
-#####################
+menu_line_even () {
+	# displays an even (cyan) line of a menu line with $2 as an indicator in [] and $1 as the option
+	printf "$c_blue[$c_cyan%c$c_blue] $c_cyan%s$c_reset%s\n" "$2" "$1"
```

Comment:
this is a lil bit of a shell check thing. The shell check confuses this with arrays in bash since there is a `$variable[...]` variable and bracket. So to avoid this you could do `${variable}[`

Reply:
Thanks, I'm new to shellcheck so I don't know the ins and outs of it.

Reply:
> this is a lil bit of a shell check thing. The shell check confuses this with arrays in bash since there is a `$variable[...]` variable and bracket. So to avoid this you could do `${variable}[`

100% correct. Do you think I should raise an issue for that on their repo?

Reply:
It backfired, because it put the `{}`-s into the string...